### DISCUSSION TITLE
HR4 ocean 1x1 grib2 not archived

### DISCUSSION THREAD

**Discussion by jiandewang** [NOAA-EMC/global-workflow#2963] [General]
Sulagna chekced the HR4 ocean_grib2.tar on HPSS and found out that it only contains 0.25 degree ocean grib2 files, no 1x1 files. Is the 1x1 files being ignored intentially in latest g-w ?
from my HR4 run, I can see these 1x1 grib2 files are being generated

from HR4 archiving run log:
- archive     : ( {'name': 'OCEAN_GRIB2', 'target': '/NCEPDEV/emc-climate/5year/Jiande.Wang/WCOSS2/HR4/2020090100/ocean_grib2.tar', 'required': ['gfs.20200901/00/products/ocean/grib2/0p25/gfs.ocean.t00z.0p25.f006.grib2', 'gfs.20200901/00/products/ocean/grib2/0p25/gfs.ocean.t00z.0p25.f006.grib2.idx
...........
..........

no 1x1 files being included here

**Comment by aerorahul** [comment]
My guess is it was done to save tape storage space.
Is there a reason 1-degree grib files need to be archived as well?  Is the 0.25-deg data not sufficient?
If we need to archive 1-degree grib output, we can add a couple of lines [here](https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/archive/ocean_grib2.yaml.j2)

**Comment by jiandewang** [comment]
after discussion with user, we feel that archiving 1x1 file is not a must as users can massage 0.25 files to fit for their own purpose.
